### DISCUSSION TITLE
1.16.5 Support

### DISCUSSION THREAD

**Discussion by Canina-X** [TeamGalacticraft/Galacticraft#204] [Q&A] [answered]
Hello, this is a awesome mod! But it has no 1.16.5 support, that's really sad. If you can make this to 1.16.5 That would be awesome!

**Comment by JustPyrrha** [comment, marked as answer]
While Galacticraft 5 is still in pre-alpha/alpha development we'll only be developing for the latest stable version of Minecraft. \
This will change once we reach stable 5.0 but we are yet to decide internally on what game versions we're going to support after that.

**Reply by Canina-X** [reply]
Alright, i hope that 1.16.5 will come out. Have a nice day
